src/main/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -88,7 +90,8 @@ class StockMovement implements Validateable{
 
     static transients = [
             "electronicType",
-            "pendingApproval"
+            "pendingApproval",
+            "displayStatus"
```

Comment:
there is nothing wrong in that, but just a note: we don't need to add transients in Grails 3 anymore. Not having a setter doesn't break the build anymore.
source: https://docs.grails.org/latest/ref/Domain%20Classes/transients.html

> As of Grails 2.0 if there is only a getter or only a setter method, you don’t need to declare the property name of the method in the transients list.